src/components/ThreadEnvelope.vue

```diff
@@ -591,7 +592,7 @@ export default {
 		left: 4px;
 	}
 	.envelope--header.list-item-style {
-		border-radius: 16px;
+		border-radius: 16px 16px 0 0;
```

Comment:
Doesn't this look a bit off when hovering/focusing the heading? Or you prefer it like that?

Reply:
Yes, earlier, when the NC Mail design was different, this fix would have been useful. But now there is no background in this element. I'll clean it up.